src/features/Dashboard/components/RecentTransactionList.tsx

```diff
@@ -126,12 +131,35 @@ export default function RecentTransactionList({ accountsList, className = '' }:
           address: recipient,
           networkId
         };
+        if (valueTransfers.length == 0) {
+          valueTransfers.push({
+            asset: baseAsset,
+            to,
+            from,
+            amount: fromTokenBase(bigify(value), DEFAULT_ASSET_DECIMAL).toString(),
+            isNFTTransfer: false
+          } as IFullTxHistoryValueTransfer);
+        }
         const entryConfig = constructTxTypeConfig(txTypeMetas[txType] || { type: txType });
+        const sentValueTransfers = valueTransfers.filter((t) => accountsMap[t.from.toLowerCase()]);
```

Comment:
We should check that the account is on the same network

Reply:
Goes for multiple places in this file

Reply:
I changed the map to use uuid generated by `generateDeterministicAddressUUID` instead of address.